pkg/api/search.go

```diff
@@ -39,7 +39,7 @@ func (hs *HTTPServer) Search(c *models.ReqContext) response.Response {
 		}
 	}
 
-	dbUIDs := c.QueryStrings("dashboardUID")
+	dbUIDs := c.QueryStrings("dashboardUIDs")
```

Comment:
even though `dashboardUID` is not documented, people may still use it; therefore I would suggest better keep both and in a future release deprecate it.